eras/conway/impl/src/Cardano/Ledger/Conway/Era.hs

```diff
@@ -144,5 +145,6 @@ type instance EraRule "SNAP" (ConwayEra c) = ShelleySNAP (ConwayEra c)
 type instance EraRule "TICK" (ConwayEra c) = ShelleyTICK (ConwayEra c)
 
 type instance EraRule "POOL" (ConwayEra c) = ShelleyPOOL (ConwayEra c)
+type instance EraRuleFailure "POOL" (ConwayEra c) = ShelleyPoolPredFailure (ConwayEra c)
```

Comment:
This instance already exists here: https://github.com/IntersectMBO/cardano-ledger/blob/f0d71456e5df5a05a29dc7c0ac9dd3d61819edc8/eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/Pool.hs#L101

It is just an orphan and the module needs to be imported to `Cardano.Ledger.Conway.Rules` as so: `import Cardano.Ledger.Conway.Rules.Pool ()`
```suggestion
```